### PR TITLE
fixing code formatting

### PR DESCRIPTION
**Pull request by mairaw**
Was looking at this topic (https://docs.microsoft.com/en-us/dotnet/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer#adding-startup-parameters) and noticed the code was all so badly formatted that it wasn't even readable.

### CODE REVIEW

File: docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md [hunk 1]
```diff
@@ -309,37 +309,88 @@ This article demonstrates how to create a simple Windows Service application in
...
+static void Main(string[] args)
+{
+    ServiceBase[] ServicesToRun = new ServiceBase[] { new MyNewService(args) };
+    ServiceBase.Run(ServicesToRun);
+}
+```  
+  
+```vb
+Shared Sub Main(ByVal cmdArgs() As String)
+    Dim ServicesToRun() As System.ServiceProcess.ServiceBase = New System.ServiceProcess.ServiceBase() {New MyNewServiceVB(cmdArgs)}         System.ServiceProcess.ServiceBase.Run(ServicesToRun)
```
Comment (GuardRex) [inline docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md]: Return needed here.

**Review by guardrex — CHANGES_REQUESTED** [review]
Looks like one return needed in a vb example.

File: docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md [hunk 2]
```diff
@@ -309,37 +309,88 @@ This article demonstrates how to create a simple Windows Service application in
...
+    Dim logName As String = "MyNewLog"
+    If (cmdArgs.Count() > 0) Then
+        eventSourceName = cmdArgs(0)
+    End If
+    If (cmdArgs.Count() > 1) Then
+        logName = cmdArgs(1)
+    End If
+    eventLog1 = New System.Diagnostics.EventLog()
+    If (Not System.Diagnostics.EventLog.SourceExists(eventSourceName)) Then
+        System.Diagnostics.EventLog.CreateEventSource(eventSourceName, logName)
```
Comment (rpetrusha) [inline docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md]: There's probably a tab here that should be replace with whitespace -- this line is indented much too far.

Comment (mairaw) [inline docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md]: I don't see this @rpetrusha. It looks the same as others. Or am I missing something?
image

**Comment by mairaw** [comment]
This should be ready for another review after build finishes @GuardRex.

**Review by guardrex — APPROVED** [review]

**Comment by GuardRex** [comment]
@mairaw Cool ... but I'm putting the interpolated strings into one of the PR's I'm working. That's an option here, too. ... Either way, it's :shipit: good! :smile:

File: docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md (hunk 2 above)
Comment (rpetrusha) [inline docs/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md]: I think it was the `MyNewService` constructor in C#. The `eventLog1` assignment is intended too far, and the call to `CreateEventSource` is also indented too far in the [published page](https://review.docs.microsoft.com/en-us/dotnet/framework/windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer?branch=pr-en-us-2483).

**Pull request merged**